AirCasting/Settings/SettingsView.swift

```diff
@@ -9,29 +9,58 @@ import SwiftUI
 import AirCastingStyling
 
 struct SettingsView: View {
+    var viewModel: SettingsViewModel
     let urlProvider: BaseURLProvider
     let logoutController: LogoutController
+    let sessionContext = CreateSessionContext()
     @State private var showBackendSettings = false
+    @State private var startSDClear = false
+    @State private var BTScreenGo = false
+    @State private var LocationScreenGo = false
+    private var SDClearingRouteProcess = true
     @EnvironmentObject var userSettings: UserSettings
+    @EnvironmentObject var bluetoothManager: BluetoothManager
     
-    init(urlProvider: BaseURLProvider, logoutController: LogoutController) {
+    init(urlProvider: BaseURLProvider, logoutController: LogoutController, viewModel: SettingsViewModel) {
         let navBarAppearance = UINavigationBar.appearance()
         navBarAppearance.largeTitleTextAttributes = [.foregroundColor: UIColor(Color.darkBlue),
                                                      .font: Fonts.navBarSystemFont]
         self.urlProvider = urlProvider
         self.logoutController = logoutController
+        self.viewModel = viewModel
     }
     
     var body: some View {
         NavigationView {
             Form {
                 signOutSection
                 settingsSection
+                #if BETA || DEBUG
+                Section() {
+                    navigateToAppConfigurationButton
+                }
+                #endif
                 appInfoSection
             }
             .listStyle(GroupedListStyle())
             .navigationBarTitle(Strings.Settings.title)
         }
+        .fullScreenCover(isPresented: $startSDClear) {
+            CreatingSessionFlowRootView {
+                SDRestartABView(viewModel: SDRestartABViewModelDefault(urlProvider: urlProvider, isSDClearProcess: SDClearingRouteProcess), creatingSessionFlowContinues: $startSDClear)
+            }
+        }
+        .fullScreenCover(isPresented: $LocationScreenGo) {
```

Comment:
This only works for ios15 unfortunately, so we need to check if it's ios15 or not and include that other implementation as well